lib/cisco_node_utils/cmd_ref/snmp_server.yaml

```diff
@@ -35,13 +35,17 @@ packet_size:
   default_value: 1500
 
 protocol:
-  config_get: "show snmp"
-  config_get_token: "protocol_status"
+  kind: boolean
+  auto_default: false
```

Comment:
@saichint Glenn brings up a good point.  When you use `auto_default`, `config_get()` will return 'nil' instead of the `default_value` when the `config_get_token` produces no match.  Is this what you want?